extensions/amp-ad-network-adsense-impl/0.1/adsense-a4a-config.js

```diff
@@ -75,11 +75,13 @@ export const URL_EXPERIMENT_MAPPING = {
 /**
  * @param {!Window} win
  * @param {!Element} element
+ * @param {!boolean} useRemoteHtml
  * @returns {boolean}
  */
-export function adsenseIsA4AEnabled(win, element) {
+export function adsenseIsA4AEnabled(win, element, useRemoteHtml) {
+  dev().assert(!useRemoteHtml, 'Adsense should never use remote.html');
   if (!isGoogleAdsA4AValidEnvironment(win) ||
-      !element.getAttribute('data-ad-client')) {
+      !element.getAttribute('data-ad-client') || useRemoteHtml) {
```

Comment:
I would make it first given its the simplest to evaluate (perhaps the compiler does this)